src/main/java/apoc/export/util/FileUtils.java

```diff
@@ -23,6 +25,27 @@ public static CountingReader readerFor(String fileName) throws IOException {
         return new CountingReader(file);
     }
 
+    public static String changeFileUrlIfImportDirectoryAssigned(String url){
+        if(isFile(url) && ApocConfiguration.isEnabled("import.file.use_neo4j_config")) {
+            // check 'dbms.security.allow_csv_import_from_file_urls' and 'dbms.directories.import' parameters
+            boolean allowCsvImportFromFile = Util.toBoolean(ApocConfiguration.list().get("dbms.security.allow_csv_import_from_file_urls"));
```

Comment:
Can we use `ApocConfiguration.isEnabled` ? `Util.toBoolean(ApocConfiguration.list().get("dbms.security.allow_csv_import_from_file_urls"));`

Perhaps it's best to transform the two settings into apoc-config 

We could turn the white-list into a map that has as key the neo key and as value the new apoc-config-name and then copy the actual config value with the new name into the config map. 

then we could use: `ApocConfiguration.isEnabled("import.file.allow_read_from_filesystem")`